db/schema.rb

```diff
@@ -21,7 +21,7 @@
     t.datetime "updated_at", null: false
   end
 
-  create_table "dock_counts", force: :cascade do |t|tu
+  create_table "dock_counts", force: :cascade do |t|
```

Comment:
The dock_counts table and the installation_dates table are a little overkill here. I think you can store these values somewhere else, like the stations table.